NOTES.md

```diff
@@ -1,3 +1,57 @@
+# Release 2.2.4 (Modern Times)
+
+Released on 2016-02-01
+
+Bug fix release
```

Comment:
You may want to clarify that this release is strongly recommended for upgrade due to the (exceedingly unlikely, but still possible) clustering bug.